src/helpers/getTvUtils.ts

```diff
@@ -151,6 +160,45 @@ const getTvRating = (arr: TvDetailsData) => {
     return 'No rating available';
 };
 
+/**
+ * Provides more details of an individual season
+ * @param showId
+ * @param season_num
+ * @returns {Promise<SeasonDetails>}
+ */
+const getTvSeasonDetails = async (showId: number, season_num: number): Promise<SeasonDetails> => {
+    const response = await fetch(
+        `https://api.themoviedb.org/3/tv/${showId}/season/${season_num}?api_key=${import.meta.env.VITE_MOVIEDB_KEY}`
+    );
+    if (!response.ok) {
+        LOG.error('Fetch request failed with a status of ' + response.status);
+    }
+    const data: SeasonDetails = await response.json();
+    return data;
+};
+
+/**
+ * Provides additional Episode details that are not provided by Season Details
+ * @param showId
+ * @param season_num
+ * @param episode_num
+ * @returns {Promise<EpisodeDetails>}
+ */
+const getTvEpisodeDetails = async (
+    showId: number,
+    season_num: number,
+    episode_num: number
+): Promise<EpisodeDetails> => {
+    const response = await fetch(
+        `https://api.themoviedb.org/3/tv/${showId}/season/${season_num}/episode/${episode_num}?api_key=${import.meta.env.VITE_MOVIEDB_KEY}&append_to_response=images,videos,credits`
```

Comment:
We could make this a constant as well. I think this is what I would call `TMDB_BASE_PATH` and the other the image path.